requirements.txt

```diff
@@ -0,0 +1,9 @@
+# pip install -r requirements.txt
+flask
+flask_login
+flask_sqlalchemy
+<<<<<<< HEAD
```

Comment:
It looks like you have a merge conflict in your requirements.txt file.  You don't want to be pushing merge conflicts into the main branch.  If this code was actually used to install the requirements it would fail.

Reply:
Good catch. Thanks for pointing it out!